app/views/claims/organisations/index.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl"><%= t("organisations") %></h1>
+
+    <% if @schools.any? %>
+      <ul class="govuk-list">
+        <% @schools.each do |school| %>
+          <li>
+            <%= govuk_link_to school.name, request.env["PATH_INFO"], class: "govuk-link--no-visited-state" %>
```

Comment:
[From the `govuk_link_to` docs](https://govuk-components.netlify.app/helpers/link/#other-link-styles), it looks like we can pass an option to turn off visited state. This might be a cleaner way to do it rather than manually setting the class attribute.

```suggestion
            <%= govuk_link_to school.name, request.env["PATH_INFO"], no_visited_state: true %>
```